src/main/java/com/github/dockerjava/api/command/UpdateContainerCmd.java

```diff
@@ -0,0 +1,68 @@
+package com.github.dockerjava.api.command;
+
+import com.github.dockerjava.api.model.UpdateContainerResponse;
+
+import javax.annotation.CheckForNull;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public interface UpdateContainerCmd extends SyncDockerCmd<UpdateContainerResponse> {
+    @CheckForNull
+    String getContainerId();
+
+    @CheckForNull
+    public Integer getBlkioWeight();
+
+    public UpdateContainerCmd withBlkioWeight(Integer blkioWeight);
+
+    public UpdateContainerCmd withContainerId(String containerId);
```

Comment:
It would be better to deny `null` here (by adding a non-null check in impl and annotate with @Nonnull).  It doesn't make sense to pass `null`. What should be done when container id is `null` in exec?